cmd/juno/juno.go

```diff
@@ -118,14 +118,14 @@ const (
 	defaultGwTimeout                = 5 * time.Second
 	defaultCorsEnable               = false
 
-	configFlagUsage                       = "The yaml configuration file."
-	logLevelFlagUsage                     = "Options: debug, info, warn, error."
+	configFlagUsage                       = "The YAML configuration file."
+	logLevelFlagUsage                     = "Options: debug, info, warn, error, trace."
```

Comment:
Looks like it's ordered from the most verbose to least. I would put trace in the first place.

Reply:
Oh, interesting. I actually followed the order here: https://github.com/NethermindEth/juno/blob/3283b79ef8eae76c374eab6c5c3b7e3b76a43726/utils/log.go#L14-L17

Reply:
yep, let's update that as well